index.html

```diff
@@ -71,18 +74,39 @@ <h2>Tutorial Videos</h2>
 				<br>
 				<h4>It's all in the name!</h4>
                 <p>Condense as much as you can with your Learning Object names while keeping the essence of that object is about. This convention makes your object easier to identify during searches.</p>
 			</div>
 			<br><br>
 
+			<a name="Users"><div class="section">
+				<h2>Learning Objects Dashboard</h2>
+				<hr>
+				<p>To access your dashboard, click the drop-down for your name and select <i>Dashboard</i> in the nav bar at the top of the page. You will be redirected to the dashboard page. On this page, you can search and manage your learning objects. Learning objects can be found under either the <i>Draft</i> or <i>Released</i> tabs.</p>
+				<div class="picture">
+					<img src="./Resources/CLARK-Dashboard.png" width="1000px">
+				</div>
+				<p style="padding-top: 20px;">Under the <i>Draft</i> tab, you can create new learning objects, edit unreleased learning objects, and manage learning objects that are waiting for review, under review, in proofing, or denied. You can search by keywords or filter learning objects by status under this tab.</p>
+				<div class="picture">
+					<img src="./Resources/CLARK-Dashboard-DraftFilters.png">
+				</div>
+				<p style="padding-top: 20px;">Under the <i>Released</i> tab, you will be able to view all released learning objects. You can view and manage learning object information such as metrics and create revisions, as well as search your released learning objects.</p>
+				<div class="picture">
+					<img src="./Resources/CLARK-Dashboard-Released.png" width="1000px">
+				</div>
+			</div>
+			<br><br>
+
 			<a name="Uploading"><div class="section">
-				<h2>Uploading a Learning Object</h2>
+				<h2>Creating a New Learning Object</h2>
 				<hr>
-				<p>To create a new learning object, you must first sign into CLARK. Click on the <i>Contribute</i> dropdown which will give you the option to go to your learning object dashboard or create a new object. If you are in your dashboard, you can create a object by clicking on the <i>New</i> button.</p>
-				<p>After entering the name of your learning object, you will notice that each object also has a length. You will need to specify the length of the learning object you will be uploading. Below are the five lengths an object can have:</p>
+				<p>Navigate to your dashboard. Under the <i>Draft</i> tab, you can create a new learning object by clicking the <i>NEW +</i> button. If there are no learning objects in your <i>Draft</i> tab, you may also click the <i>CREATE A LEARNING OBJECT +</i> button. The learning object builder will open. You will first need to give your learning object a name in order to be able to add learning outcomes, which at least one is required, and to also be able to upload materials.</p>
```

Comment:
For the comments about how one outcome is required lets either add a section here or create a new section that states what exactly a learning object needs to be able to be submitted:
- 1+ valid outcomes
- Description
- Name
- 1+ contributors
- 1+ levels

We don't validate materials